app/services/forest_liana/resources_getter.rb

```diff
@@ -110,7 +110,9 @@ def association?(field)
     end
 
     def select
-      column_names = @resource.column_names.map { |name| name.to_sym }
+      column_names = @resource.column_names.map do |name|
+        "#{@resource.table_name}.#{name}".to_sym
```

Comment:
Hey @tsara27, the change you made is obviously going to the right direction but it broke the performance optimizations we implemented below. We only want to select the column names found in the request params.